client/src/pages/loginPage.tsx

```diff
@@ -0,0 +1,29 @@
+import { Box, Button, Container, Divider, FormLabel, Grid, InputLabel, TextField, Typography } from '@mui/material'
+import React from 'react'
+import logo from '../assets/SoleAI_Logo.png'
+
+const ContainerStyle = {width:1,height:"98vh",display:"flex",flexDirection:{xs:"column",sm:"column",md:"row",lg:"row"}, alignItems:"center"}
+const ImageStyle = {width:{xs:1/4,sm:1/4, md:1, lg:1} , visibility:{xs:"hidden",sm:"hidden",md:"visible",lg:"visible"}}
+const LoginPage = () => {
+  return (
+    <Container sx={ContainerStyle}>
+        <Box sx={ImageStyle}>
+            <img src={logo} style={{width:"70%",margin:"15%"}}/>
+        </Box>
+        <Box width="100%"> 
+            <Box sx={{display:'flex',flexDirection:'column',gap:3,mx:{xs:2,sm:15,md:10,lg:6}}}>
```

Comment:
styles should be in different js object for code cleanliness

Reply:
yes done